deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java

```diff
@@ -138,7 +143,12 @@ public void generateWSDL(BuildProducer<NativeImageResourceBuildItem> ressources,
         }
     }
 
-    private String getNamespaceFromPackage(String pkg) {
+    private String getNamespaceFromPackage(ClassInfo wsClassInfo) {
```

Comment:
can you change it to getNameSpaceFromClassInfo

Reply:
Sure, good point! I wanted to change the name anyway but forgot to do so.

Reply:
done